### PR TITLE
Theme: 

### PR DESCRIPTION
**Pull request by raynesmike**
use hooks for Theme to update depending of what theme were going to put
Theme
.

### CODE REVIEW

File: mobile/src/theme/apptheme/AppTheme.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    dark: false,
+    colors: {
+      primary: color.primary,
+      background: color.black,
+      card: color.primary,
+      text: color.black,
+      border: color.primary,
+      notification: color.primary,
+    },
+  },
```
Comment (lasharela) [inline mobile/src/theme/apptheme/AppTheme.ts:15]: We don't need to have dark theme

File: mobile/src/theme/apptheme/AppTheme.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  white: {
+    dark: false,
+    colors: {
+      primary: color.blue,
+      background: color.white,
```
Comment (lasharela) [inline mobile/src/theme/apptheme/AppTheme.ts:20]: we need only background here. and you have to re-assign primary as well.

File: mobile/src/theme/apptheme/AppTheme.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      card: color.primary,
+      text: color.black,
+      border: color.primary,
+      notification: color.primary,
+    },
+  },
+  white: {
+    dark: false,
+    colors: {
+      primary: color.blue,
```
Comment (lasharela) [inline mobile/src/theme/apptheme/AppTheme.ts:19]: we can replace this one

**Review by lasharela — APPROVED** [review]

**Review by Dulaya — APPROVED** [review]
Looks good to me.

**Review by mshago — APPROVED** [review]

**Review by aahenry51 — APPROVED** [review]

**Review by carizzuti — APPROVED** [review]

**Pull request merged**